src/cli.py

```
@@ -164,9 +184,68 @@ def main():
+            else:
+                model = questionary.select(
+                    "Select Google model",
+                    choices=[
+                        "gemini-2.5-flash-image",
+                    ],
+                ).ask()
+
+            asyncio.run(process_campaign(args.brief, provider, model))
+
+        elif args.command == "monitor":
+            asyncio.run(run_monitoring_agent(args.interval, args.sla_threshold))
```

Ruff (0.13.3)

Undefined name `run_monitoring_agent`

(F821)